bin/xucli

```diff
@@ -27,6 +27,12 @@ require('yargs')
       describe: 'Path to the TLS certificate of xud',
       type: 'string',
     },
+    raw: {
```

Comment:
For this option, how about calling it `json` with a `-j` flag and description `Display output in json format`. My thinking is that the "raw" responses are serialized data and that the json output is actually formatted. This makes it clear that the output will be in JSON.

Reply:
Calling it `json` instead of `raw` feels way better.